### PR TITLE
Document calling non-threadsafe primitives from threads

### PR DESCRIPTION
**Pull request by uSpike**
This also adds a `docs` environment to tox so that the documentation can easily be built locally with `tox -e docs`

### CODE REVIEW

**Review by agronholm — CHANGES_REQUESTED** [review]
Looks good, just a couple of changes.

File: tox.ini [hunk 1]
```diff
@@ -37,3 +37,7 @@ deps = mypy >= 0.900
 extras = test
 commands = mypy --install-types {posargs} src tests
 skip_install = true
+
+[testenv:docs]
+extras = doc
+commands = sphinx-build docs docs/_build
```
[the remark this answers is not in the document]
Comment (uSpike) [reply tox.ini:43]: Sure, I'll add a note there, thanks.

[the remark this answers is not in the document]
Comment (uSpike) [reply tox.ini]: sphinx-build requires an output path, so I set that as `build/`.

[the remark this answers is not in the document]
Comment (uSpike) [reply tox.ini]: I also added the contributor docs

File: docs/threads.rst [hunk 2]
```diff
@@ -53,6 +53,28 @@ If you need to call a coroutine function from a worker thread, you can do this::
 .. note:: The worker thread must have been spawned using :func:`~run_sync_in_worker_thread`
    for this to work.
 
+Calling synchronous code from a worker thread
+---------------------------------------------
+
+AnyIO primitives should not be assumed thread-safe, therefore they should not be used directly
+from worker threads.  It is important to call synchronous code using :func:`~from_thread.run_sync`
+when inside a worker thread::
```
[the remark this answers is not in the document]
Comment (uSpike) [reply docs/threads.rst]: Thanks for the suggestion, I had mostly just copied the asyncio documentation language.

**Review by agronholm — APPROVED** [review]

**Comment by agronholm** [comment]
Thanks!

**Pull request merged**